app/uk/gov/hmrc/tai/service/TrackingService.scala

```diff
@@ -57,17 +58,20 @@ class TrackingService @Inject()(
         )
 
         (haveAnyShortProcesses, haveAnyLongProcesses, filteredJournies.isEmpty, isA3WeeksJourney(successfulJournies)) match {
-          case (true, false, _, _) | (_, _, false, false) => SevenDays
+          case (true, false, _, _) | (_, _, false, false) => FifteenDays
```

Comment:
part of me thinks that an old if else would make it easier to reason about but meh

Reply:
good ol' if else